PermissionsScraper/PermissionsScraper/Helpers/ProtectedApiCallHelper.cs

```diff
@@ -23,16 +24,10 @@ internal static class ProtectedApiCallHelper
         /// <returns>The response of the call to the Web API</returns>
         internal static async Task<string> CallWebApiAsync(string webApiUrl, string accessToken)
         {
-            if (string.IsNullOrEmpty(webApiUrl))
-            {
-                throw new ArgumentNullException(nameof(webApiUrl), "Parameter cannot be null or empty");
-            }
-            if (string.IsNullOrEmpty(accessToken))
-            {
-                throw new ArgumentNullException(nameof(accessToken), "Parameter cannot be null or empty");
-            }
+            UtilityFunctions.CheckArgumentNullOrEmpty(webApiUrl, nameof(webApiUrl));
+            UtilityFunctions.CheckArgumentNullOrEmpty(accessToken, nameof(accessToken));
 
-            HttpClient httpClient = new HttpClient();
+            HttpClient httpClient = new();
             HttpRequestHeaders defaultRequestHeaders = httpClient.DefaultRequestHeaders;
             if (!defaultRequestHeaders.Accept.Any(m => m.MediaType == "application/json"))
```

Comment:
```suggestion
            if (!defaultRequestHeaders.Accept.Any(m => "application/json".Equals(m.MediaType, StringComparison.OrdinalIgnoreCase)))
```

Reply:
Why not `if (!defaultRequestHeaders.Accept.Any(m => m.MediaType.Equals("application/json", StringComparison.OrdinalIgnoreCase)))` ?

This is easier to read in my opinion.

Reply:
because the style I'm suggesting protects you against MediaType being null, the one you're suggesting doesn't